R/utilities.R

```diff
@@ -3015,6 +3020,116 @@ get_bams = function(sample,
   return(bam_details)
 }
 
+#' @title View a variant in IGV
+#'
+#' @description Load bam(s) and view the context around a mutation
+#'
+#' @details 
+#'
+#' @param this_mutation Specify the mutation of interest in MAF format.
```

Comment:
Multiple parameter descriptions seem to be missing.

Reply:
I think this is resolved now

Reply:
I think the problem here is I couldn't tell what function you were commenting on because the line numbering shown in this interface didn't match the line number in my file and the code context I see is not allowing me to find the function. What function are you commenting on?